pkg/net/net_linux.go

```diff
@@ -68,6 +69,8 @@ func nics(ctx *context.Context) []*NIC {
 		nic.MacAddress = mac
 		if etAvailable {
 			nic.Capabilities = netDeviceCapabilities(ctx, filename)
+			// Sets NIC struct fields and appends NICCapability from ethtool output
+			nic.setNicAttrEthtool(ctx, filename)
```

Comment:
Can we place the call to `setNicAttrEthtool` within the `netDeviceCapabilities` function?